core/urls.py

```diff
@@ -49,6 +52,8 @@
                 # v1 urls
                 path("rester/", include("rester.urls")),
                 path("todo/", include(todo_urls)),
+                path("logs/", logs.index_directory_log, name="logs-dir"),
```

Comment:
Logs aren't the part of standard api and should be in same level as of swagger, admin, etc. Thanks